__tests__/endpoints.test.js

```diff
@@ -352,6 +352,40 @@ describe('POST /api/articles/:article_id/comments', () => {
     });
 });
 
+describe('DELETE /api/comments/:comment_id', () => {
+    describe('204', () => {
+        test('204: deletes the comment with the comment for the given comment_id and responds with no content', () => {
+            return request(app)
+            .delete('/api/comments/1')
+            .expect(204)
+            .then(({body}) => {
```

Comment:
We could delete line 361 and 362 because the status code 204 does not "carry" a body so it would always be empty 🙂